perspective_reddit_bot/moderate_subreddit.py

```diff
@@ -165,17 +174,21 @@ def score_subreddit(creds_dict,
     try:
       if i%100 == 0 and i > 0:
         print(i)
-      if len(comment.body) > 20000:
+      original_comment = comment.body
+      comment_for_scoring = (remove_quotes(original_comment)
+                             if should_remove_quotes else original_comment)
+      if len(comment_for_scoring) > 20000:
         print('Comment too long, skipping...')
         continue
-      scores = perspective.score_text(comment.body, api_models,
+      scores = perspective.score_text(comment_for_scoring, api_models,
```

Comment:
Is it worth checking that the comment_for_scoring is non-empty?

Reply:
Ah, good. point. I changed `remove_quotes` to return the original text instead, so we always send some text, if it's possible.